Index: .github/workflows/main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># cocktail sort\r\n\r\ndef cocktail_sort(nums):\r\n    for i in range(len(nums) - 1, 0, -1):\r\n        is_swapped = False\r\n\r\n        for j in range(i, 0, -1):\r\n            if nums[j] < nums[j - 1]:\r\n                nums[j], nums[j - 1] = nums[j - 1], nums[j]\r\n                is_swapped = True\r\n\r\n        for j in range(i):\r\n            if nums[j] > nums[j + 1]:\r\n                nums[j], nums[j + 1] = nums[j + 1], nums[j]\r\n                is_swapped = True\r\n\r\n        if not is_swapped:\r\n            return nums\r\n\r\n\r\nnums = [15, 37, 69, 26, 78]\r\nprint(cocktail_sort(nums))\r\nprint(yazaebalas)\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.github/workflows/main.py b/.github/workflows/main.py
--- a/.github/workflows/main.py	(revision e6e5a607183395925e9b36a2f3152889cd547794)
+++ b/.github/workflows/main.py	(date 1697813446203)
@@ -1,23 +1,23 @@
 # cocktail sort
 
-def cocktail_sort(nums):
-    for i in range(len(nums) - 1, 0, -1):
+def cocktail_sort(array):
+    for i in range(len(array) - 1, 0, -1):
         is_swapped = False
 
         for j in range(i, 0, -1):
-            if nums[j] < nums[j - 1]:
-                nums[j], nums[j - 1] = nums[j - 1], nums[j]
+            if array[j] < array[j - 1]:
+                array[j], array[j - 1] = array[j - 1], array[j]
                 is_swapped = True
 
         for j in range(i):
-            if nums[j] > nums[j + 1]:
-                nums[j], nums[j + 1] = nums[j + 1], nums[j]
+            if array[j] > array[j + 1]:
+                array[j], array[j + 1] = array[j + 1], array[j]
                 is_swapped = True
 
         if not is_swapped:
-            return nums
+            return array
 
 
-nums = [15, 37, 69, 26, 78]
-print(cocktail_sort(nums))
+array = [15, 37, 69, 26, 78]
+print(cocktail_sort(array))
 print(yazaebalas)
